pkg/compute/init.go

```diff
@@ -108,6 +110,19 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	undoStack := common.NewUndoStack()
 	defer func() { undoStack.RunIfError(out, err) }()
 
+	var service *fastly.Service
+	if c.serviceID != "" {
+		service, err = c.Globals.Client.GetService(&fastly.GetServiceInput{
```

Comment:
The rest of this file assumes that the version is `1` but that might not be true. We may even need to clone an active service version. Thoughts?

Reply:
😱 Great catch! Yes, this is a very big oversight on my behalf, will fix.